docs/configuring-polaris-for-production.md

```diff
@@ -55,6 +55,13 @@ Be sure to secure your metastore backend since it will be storing credentials an
 
 To use [EclipseLink](https://eclipse.dev/eclipselink/) for metastore management, specify the configuration `metaStoreManager.conf-file` to point to an [EclipseLink `persistence.xml` file](https://eclipse.dev/eclipselink/documentation/2.5/solutions/testingjpa002.htm). This file, local to the Polaris service, will contain information on what database to use for metastore management and how to connect to it.
 
+> [!IMPORTANT]
+> EclipseLink requires
+> 1. Building the JAR for the EclipseLink extension
+> 2. Setting the `eclipseLink` gradle property to `true`.
+>
+> This can be achieved by setting `eclipseLink=true` in the `gradle.properties` file, or by passing the property explicitly while building all JARs, e.g.: `./gradlew -PeclipseLink=true clean assemble`
```

Comment:
I'd like to merge this -- can we also quickly add a link to the new [metastores](https://github.com/apache/polaris/blob/99e7e692d3e6d27edc7064dbec0503860705b024/site/content/in-dev/unreleased/metastores.md?plain=1#L20) doc?

Reply:
@eric-maynard I just synced my fork and it brought in a link to the metastores doc right above this block. Ok for you?